### PR TITLE
C2DEVEL-14659: Add tag name from last recovery point to protected resource model

### PR DESCRIPTION
**Pull request by RudeZwloki7**
Closes issue: C2DEVEL-14659

### CODE REVIEW

File: botocore/data/backup/2018-11-15/service-2.json [hunk 1]
```diff
@@ -1949,6 +1949,10 @@
         "LastBackupTime":{
           "shape":"timestamp",
           "documentation":"<p>The date and time that a resource was last backed up, in Unix format and Coordinated Universal Time (UTC). The value of <code>LastBackupTime</code> is accurate to milliseconds. For example, the value 1516925490.087 represents Friday, January 26, 2018 12:11:30.087 AM.</p>"
+        },
+        "LastTagName":{
+          "shape":"string",
+          "documentation":"<p>The tag name of resource retrieved from last recovery point.</p>"
```
Comment (kinazarov1999) [inline botocore/data/backup/2018-11-15/service-2.json:1955]: `the last recovery point.`

**Review by kinazarov1999 — CHANGES_REQUESTED** [review]

**Comment by funkmajer** [comment]
TIW

**Comment by Sharpeye90** [comment]
@RudeZwloki7 rebase pls

**Comment by RudeZwloki7** [comment]
@Sharpeye90 done

**Pull request merged**